### PR TITLE
[DOC] Minor updates to software paper for JOSS review

### PR DESCRIPTION
**Pull request by emdupre**
### Summary

Relates to https://github.com/openjournals/joss-reviews/issues/6434

### Describe your changes

Minor updates to the software paper accompanying the JOSS submission for MapReader for readability and formatting. 

### Checklist before assigning a reviewer (update as needed)

- [ ] Self-review code
- [ ] Ensure submission passes current tests
- [ ] Add tests
- [ ] Update relevant docs

### Reviewer checklist

Please add anything you want reviewers to specifically focus/comment on.

- [ ] Everything looks ok?

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -49,9 +49,9 @@ bibliography: paper.bib
...
 
-MapReader's approach was inspired by methods in biomedical imaging, which were adapted for use by historians, and it is suitable for a wide range of applications in image analysis: it has, for example, been applied to an image classification problem in plant phenotype research [@Corcoran]. This cross-pollination between the humanities and the natural sciences was made possible by the open and reproducible research methods at the heart of MapReader.
+MapReader's approach was inspired by methods in biomedical imaging and adapted for use by historians, and it is suitable for a wide range of applications in image analysis: it has, for example, been applied to an image classification problem in plant phenotype research [@Corcoran]. This cross-pollination between the humanities and the natural sciences was made possible by the open and reproducible research methods at the heart of MapReader.
```
Comment (kmcdono2) [inline paper/paper.md:54]: I prefer to leave this in the original form

File: paper/paper.md [hunk 2]
```diff
@@ -62,14 +62,14 @@ In this paper, we present the MapReader release at the conclusion of the Living
...
+MapReader is among the first end-to-end pipelines for processing historical maps and other images that are designed to lower barriers to experimenting with computer vision in answering research questions about large image datasets. Other projects are emerging which perform similar research tasks with the visual content in historical map collections [@Petitpierre; @Combes]. Other tools, like the Distant Viewing Toolkit [@Arnold], address similar needs for other kinds of media. As part of a collaboration between Machines Reading Maps and the David Rumsey Historical Map Collection, the Knowledge Computing Lab released mapKurator [@mapkurator], which takes map image input and creates geojson files of all text on each map.
```
Comment (kmcdono2) [inline paper/paper.md:72]: I am going to make an amendment in the paper to update the final sentence. (I will do this outside of this review.)

**Review by kmcdono2 — CHANGES_REQUESTED** [review]

**Comment by kallewesterling** [comment]
@kmcdono2 @rwood-97 Is this the PR that's blocking the JOSS paper from going through?

**Comment by emdupre** [comment]
Apologies, I hadn't realized I didn't push the reversion on the suggested rephrasing.

I've also incorporated the update to the unpublished Corcoran reference.

**Review by rwood-97 — APPROVED** [review]

**Pull request merged**